.coveragerc

```diff
@@ -654,6 +654,7 @@ omit =
     homeassistant/components/whois/sensor.py
     homeassistant/components/wink/*
     homeassistant/components/wirelesstag/*
+    homeassistant/components/withings/*
```

Comment:
Don't exclude modules from coverage if their coverage is good. The `config_flow.py` module must not be excluded and must have 100% coverage.

Reply:
Opps. I overlooked this, I thought it was inclusions.